examples/13_two_tensor_op_fusion/device/b2b_gemm.h

```diff
@@ -295,12 +313,12 @@ class B2bGemm {
   static size_t get_workspace_size(Arguments const &args) {
 
     size_t bytes = 0;
-      
+
     // Determine grid shape
     ThreadblockSwizzle threadblock_swizzle;
 
     cutlass::gemm::GemmCoord tiled_shape = threadblock_swizzle.get_tiled_shape(
-      args.problem_size_0, 
+      args.problem_size_0,
       {ThreadblockShape0::kM, ThreadblockShape0::kN, ThreadblockShape0::kK},
       args.split_k_slices);
```

Comment:
We could use batch_count to initialize this, refer to https://github.com/NVIDIA/cutlass/blob/a31b43b3f3cb609c6ed0b30ca330d766291e84fb/include/cutlass/gemm/kernel/params_universal_base.h#L144.